src/main/java/com/mwo/group2/Student.java

```diff
@@ -8,14 +8,14 @@
 	private String group;
 	private String labGroup;
 	private Status status;
-	private List<Grades> oceny;
+	private List<Grades> grades;
 
-	public Student(short year, String group, String labGroup, Status status, List<Grades> oceny) {
+	public Student(short year, String group, String labGroup, Status status, List<Grades> grades) {

```

Comment:
dodaj tez pusty konstruktor - na chwile obecna nie jest potrzebny, ale Java Reflection bedzie uzywac jak bedziemy wykorzystywac Spring & Hibernate. To nie Twoj blad, tlyko takie info na przyszlosc.